feature-libs/cart/base/assets/translations/en/cart.json

```diff
@@ -9,7 +9,7 @@
     "description": "Description",
     "item": "Item",
     "itemPrice": "Item price",
-    "quantity": "Qty",
+    "quantity": "Quantity",
```

Comment:
We should be careful about changing or removing properties already in use due to [breaking changes](https://sap.github.io/spartacus-docs/breaking-changes/#avoiding-breaking-changes) (even translations). 

Lets create a new property for "Quantity" to avoid making the breaking change and hide any use of the translation behind a feature toggle. You can do this in either the template or the service. It is similar to how you have already done with styles.

In template example:
```
// feature off
<ng-container *cxFeature="'!a11yQTY2Quantity'">
// show old content or translation
...
// feature on
<ng-container *cxFeature="'a11yQTY2Quantity'">
// show new
```

In service:
```
someMethod() {
  ...
  if( this.featureConfigService.isEnabled('a11yQTY2Quantity') {
    // Do something when feature flag is enabled
  }
  ...
}

Reply:
I've updated it.